PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,18 @@
 ## Description
 
-*(Instructions: replace text in this and all sections with your own text)*
+Provide a detailed description of what this PR does. What problem does it fix? What new capability does it add?
 
-Provide a detailed description of what this PR does.
+## Issue(s) addressed
 
-What problem does it fix? What new capability does it add?
-
-[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.
-
-### Issue(s) addressed
-
-Link the issues to be closed with this PR
-- fixes #<issue_number>
-
-If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)
-
-## Acceptance Criteria (Definition of Done)
-
-What does it mean for this PR to be finished?
+Fixes #<issue_number>
 
 ## Dependencies
 
-If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
-Waiting on the following PRs:
-- [ ] waiting on JCSDA/eckit/pull/<pr_number>
-- [ ] waiting on JCSDA/atlas/pull/<pr_number>
+List the other PRs that this PR is dependent on, including updates to documentation:
```

Comment:
I feel the prompt to remember to update documentation might be more appropriate in the "Impact" section than "Dependencies".

Reply:
I agree. A check box or section on documentation updates would be useful.

Reply:
@svahl991, @climbfuji: what do you think would be better? A separate _Documentation_ section? Or put in the _Impact_?

Reply:
Maybe take a look at a PR template from NOAA-GFDL that uses check boxes for a few basic checks: https://github.com/NOAA-GFDL/FMS/pull/1224

Reply:
> @svahl991, @climbfuji: what do you think would be better? A separate _Documentation_ section? Or put in the _Impact_?

I'd be OK with either. Maybe I lean a little toward putting it in "Impact" just to keep the number of template sections minimal.

Reply:
I've added a checklist as suggested by Dom, I'll re-request all reviews.